### ISSUE TITLE
Doesn't show up in extensions on macOS Catalina

### ISSUE THREAD

**Issue by melgu** [Bouke/SwiftInitializerGenerator#23]
The `sudo /usr/libexec/xpccachectl` command returns `command not found`.

**Comment by Xopoko** [comment]
Same problem. But found answer here: https://stackoverflow.com/questions/47690605/script-xpccachectl-not-found

**Comment by Bouke** [comment]
I don't think you need to run `xpccachectl` at all; it isn't needed since El Capitan. @Xopoko did you verify that this extension works on the macOS Catalina beta?

**Comment by Xopoko** [comment]
Yes, i confirm.

**Comment by innocarpe** [comment]
Is it working at Xcode 11.3.1? I followed all the process in README.me but I can't see anything in `Editor` pane.
